src/Common/src/Interop/Linux/System.Net.Security.Native/Interop.Ntlm.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurity
+    {
+        // The following constant is used in calculation of NTOWF2
+        // ref: https://msdn.microsoft.com/en-us/library/cc236700.aspx
+        public const int MD5DigestLength = 16;
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurity_HeimNtlmFreeBuf")]
+        internal static extern int HeimNtlmFreeBuf(IntPtr bufferHandle);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurity_ExtractNtlmBuffer")]
+        internal static extern int ExtractNtlmBuffer(SafeNtlmBufferHandle data, byte[] buffer, int capacity, int offset);
```

Comment:
Signature parity: capacity and offset are declared here as signed integers, in the native code as unsigned integers.